src/components/side_panel/conditional_formatting/cf_editor/cf_editor.ts

```diff
@@ -311,6 +319,10 @@ export class ConditionalFormattingEditor extends Component<Props, SpreadsheetChi
           operator: "gt",
         },
       },
+      dataBar: {
+        type: "DataBarRule",
+        color: 0xffffff,
```

Comment:
if the default is white, the user is forced to always choose another color wich is an extra step. Can we have a nice light green as default color ?